hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSink.java

```diff
@@ -243,14 +250,26 @@ public void replicateEntries(List<WALEntry> entries, final CellScanner cells,
         LOG.debug("Finished replicating mutations.");
       }
 
-      if (bulkLoadHFileMap != null && !bulkLoadHFileMap.isEmpty()) {
-        LOG.debug("Started replicating bulk loaded data.");
-        HFileReplicator hFileReplicator =
-            new HFileReplicator(this.provider.getConf(this.conf, replicationClusterId),
+      if(bulkLoadsPerClusters != null) {
+        for (List<String> clusterIds : bulkLoadsPerClusters.keySet()) {
```

Comment:
Is there a reason String is used for clusterId? 
I see UUID used elsewhere in this class. like:
List<UUID> clusterIds = new ArrayList<>(entry.getKey().getClusterIdsList().size());

Reply:
That came up as well on the separate PR I created for master branch. Pasting my explanation made there:

> HRegionServer already keeps a String clusterId (not UUID), which I'm using to check against the incoming bulk load request. Converting the request List to List when clusterId is already represented as String didn't seem to give much gain here. Maybe we should raise a separate jira to convert all existing String representations of cluster id into UUID, and including this one as part of this work?

We agreed with that suggestion to open a separate jira to address replacing existing String clusterId to UUID.